Doc/library/calendar.rst

```diff
@@ -185,37 +185,42 @@ it's the base calendar for all computations.
 
       Note that the length of this list must be seven items.
 
+   .. versionadded:: 3.7
```

Comment:
The `.. versionadded:: 3.7` should go *inside* the `.. attribute::` block (after the description), i.e.

```
    .. attribute:: cssclass_noday
  
        The CSS class for a weekday occurring in the previous or coming month.

        .. versionadded:: 3.7
```

Note that you can create the documentation yourself to check the result via `make html` in the `Doc` directory (you need Sphinx for that)